komodo/release_transpiler.py

```diff
@@ -204,20 +204,31 @@ def main():
     matrix_parser.add_argument(
         "--release-folder",
         required=True,
+        type=dir_path,
         help="Folder with existing release file (default: None)",
     )
     matrix_parser.add_argument(
         "--override-mapping",
         required=True,
+        type=check_if_valid_file,
         help="File containing explicit matrix packages (default: None)",
     )
+
+    def checkValidPythonVersion(python_versions: str) -> list:
+        output_list = []
+        for item in python_versions.split(","):
+            if re.match(r"^[2,3](\.\d+)?$", item) is None:
+                raise TypeError(item)
+            output_list.append(item)
+        return output_list
+
     matrix_parser.add_argument(
         "--py_coords",
         help="""Comma delimitated list of python versions to be combined,
         for example, "3.6,3.8" (without spaces).
         If None, the release files in release-folder will be used to imply
         the versions to combine.""",
-        type=lambda s: re.split(",", s),
+        type=checkValidPythonVersion,
```

Comment:
maybe rename to `valid_python_version_list`, avoiding CamelCase and in line with the suggestion above. 

It is also an alternative to go fully explicit: `comma_delimited_python_versions`

Reply:
I agree, the function name would be way more descriptive as comma_delimited_python_versions.